doc/changelog/1.22.0-changelog.rst

```diff
@@ -0,0 +1,694 @@
+
+Contributors
+============
+
+A total of 150 people contributed to this release.  People with a "+" by their
+names contributed a patch for the first time.
+
+* @DWesl
+* @Illviljan
+* Aaron Meurer
+* Adrian Gao +
+* Ahmet Can Solak +
+* AhmetCanSolak +
+* Alban Colley +
+* Alessia Marcolini +
+* Amit Kumar +
+* Andrew Watson +
+* Anirudh Dagar +
+* Ankit Dwivedi +
+* Antony Lee
+* Arryan Singh +
+* Arun +
+* Arushi Sharma +
+* Bas van Beek
+* Carl Johnsen +
+* Carl Michal +
+* Charles Harris
+* Chiara Marmo
+* Chunlin Fang
+* Clément Robert +
+* Constanza Fierro
+* Damien Caliste
+* Daniel Ching
+* David Okpare +
+* Derek Huang +
+* Developer-Ecosystem-Engineering +
+* Dima Pasechnik
+* Dimitri Papadopoulos +
+* Dmitriy Fishman +
+* Elias Koromilas +
+* Eliaz Bobadilla +
+* Elisha Hollander +
+* Eric Wieser
+* Fayas Noushad +
+* Fayyr +
+* Gagandeep Singh +
+* Ganesh Kathiresan
+* Ghiles Meddour +
+* Gregory R. Lee
+* Guo +
+* Gwyn Ciesla +
+* Henry Schreiner +
+* Himanshu +
+* Hugo Defois +
+* Hugo van Kemenade
+* I-Shen Leong +
+* ImenRajhi +
+* IrinaMaria +
+* Irit Katriel +
+* Isuru Fernando
+* Jakob Jakobson
+* Jerry Morrison +
+* Joe Marshall +
+* Johan von Forstner +
+* Jonas I. Liechti +
+* Jonas Liechti +
+* Jonathan Reichelt Gjertsen +
+* Joshua +
+* Jérome Eertmans
+* Jérôme Kieffer +
+* KIU Shueng Chuan +
+* Kenichi Maehashi
+* Kent R. Spillner +
+* Kevin Modzelewski +
+* Kevin Sheppard
+* MalikIdreesHasanKhan +
+* Marco Aurelio da Costa +
+* MarsBarLee +
+* Marten van Kerkwijk
+* Matthew +
+* Matthew Barber +
+* Matthew Brett
+* Matthias Bussonnier
+* Matthieu Dartiailh
+* Matti Picus
+* Melissa Weber Mendonça
+* Michael McCann +
+* Mike Jarvis +
+* Mike McCann +
+* Mike Toews
+* Mukulikaa Parhari
+* NectDz +
+* Nick Pope +
+* Niels Dunnewind +
+* Niko Savola +
+* Nikola Forró
+* Niyas Sait +
+* Pamphile ROY
+* Paul Ganssle +
+* Pauli Virtanen
+* Pax +
+* Pearu Peterson
+* Peter Hawkins +
+* Peter Tillema +
+* Prathmesh Shirsat +
+* Raghuveer Devulapalli
+* Ralf Gommers
+* Robert Kern
+* Rohit Goswami +
+* Ronan Lamy
+* Ross Barnowski
+* Samyak S Sarnayak +
+* Sayantika Banik +
+* Sayed Adel
+* Sebastian Berg
+* Shriraj Hegde +
+* Sista Seetaram +
+* Stefan van der Walt
+* Stephannie Jimenez Gacha +
+* Tania Allard
+* Thomas J. Fan
+* Tim Hoffmann
+* Tom Tan +
+* Tyler Reddy
+* Vijay Arora +
+* Warren Weckesser
+* Yang Hau
+* Yashasvi Misra
+* Yuval Ofek +
+* ZHANG NA +
+* Zac Hatfield-Dodds
+* abel +
+* bbrewington +
+* bdvd +
+* czgdp1807 +
+* david-cortes +
+* default-303 +
+* dependabot[bot]
+* eskild +
+* iameskild +
```

Comment:
@iameskild want to add your real name here?

Reply:
Thanks for the ping @rgommers, looks like it's been taken care of :)